extensions/amp-selector/1.0/base-element.js

```diff
@@ -276,6 +276,6 @@ BaseElement['props'] = {
   'multiple': {attr: 'multiple', type: 'boolean'},
   'name': {attr: 'name'},
   'role': {attr: 'role'},
-  'tabindex': {attr: 'tabindex'},
-  'keyboardSelectMode': {attr: 'keyboard-select-mode'},
+  'tabindex': {attr: 'tabindex', media: true},
```

Comment:
I think we need to keep `tabindex` as close to the Web standard as we can. As such, I don't think we can allow media here. 

Also, shouldn't it be `tabIndex` (camel-case?)

Reply:
This was my line of reasoning for tabindex: If someone is using `media` attribute in AMP layer or otherwise conditionally showing some elements and not others based on viewport size, that could potentially also impact the focus order the elements should be done in, so maybe the author would want to customize this as well. But I don't feel strongly, I can remove for now.

Reply:
That certainly makes sense. And I think what you really are saying is that "tabindex feels like it really should be a CSS property and not a DOM attribute". But I think we have to stay close to Web spec on this one.

Reply:
What do you mean in particular about staying close to Web spec?

Reply:
I mean that Web spec already defines `tabindex` attribute with a specific expectation of values. I don't think we can change that.

Reply:
oooh gotcha, good point! removed

Reply:
Also, the prop name still needs camel-case, right?

Reply:
Yup, this is now: `'tabIndex': {attr: 'tabindex'},`